Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:12.18-alpine

```

Comment:
consider using bitnami nodejs as that one is the most trustworthy image

Reply:
also, use Docker builder pattern, try googling about how to reduce weight of the docker image